examples/distance.c

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2017 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Example program that calculates the distance in kilometers between two
+ * hexagon indices.
+ */
+
+// mean Earth radius
+#define R 6371.0088
+
+#include <h3/h3api.h>
+#include <math.h>
+#include <stdio.h>
+
+double dist(double th1, double ph1, double th2, double ph2) {
```

Comment:
Add a comment for this function? What are the parameters, is it Haversine distance, etc?

Reply:
done!